doc/spec/estimation/forest.rst

```diff
@@ -94,8 +94,8 @@ a multi-class classification model and should support :code:`predict_proba`.
 For more details on the input parameters of the orthogonal forest classes and how to customize
 the estimator checkout the two modules:
 
-- :py:class:`~econml.ortho_forest.DiscreteTreatmentOrthoForest`
-- :py:class:`~econml.ortho_forest.ContinuousTreatmentOrthoForest`
+    - :py:class:`~econml.ortho_forest.DiscreteTreatmentOrthoForest`
```

Comment:
These two lines don't need to be further indented; the problem was the lack of a blank line above them, which I fixed with a previous checkin; however I haven't pushed a new copy of the documentation to the website because I'm waiting for our v0.2 release.